ci/kubetest/test_clickhouse_persistence_disabled.py

```diff
@@ -0,0 +1,30 @@
+import pytest
```

Comment:
Now that we have actual kubetest, could we remove the `test_something.py`?

Reply:
Fair point and I have the `git rm` in another branch